### DISCUSSION TITLE
C++ extended running button disappears

### DISCUSSION THREAD

**Discussion by ycnx** [microsoft/vscode-discussions#200] [Extension Development QnA] [answered]
![image](https://user-images.githubusercontent.com/79720579/198288941-7c1c8d79-04d3-4ff4-8458-7ff1204be21f.png)
The triangle in the figure above disappears.
I don't know how to solve it, reinstalling expansion is invalid.

**Comment by gjsjohnmurray** [comment, marked as answer]
Please open an issue on the C++ extension repository. This discussion forum is currently targeted at people who are **developing** extensions.

**Reply by ycnx** [reply]
okay
